checker/build.gradle

```diff
@@ -647,25 +647,56 @@ task wpiManyTests(group: "Verification") {
 
     doFirst {
         delete("${project.projectDir}/build/wpi-many-tests-results/")
+        // wpi-many.sh is run in skip mode so that logs are preserved, but
+        // we don't actually want to skip previously-failing tests when we
+        // re-run the tests locally.
+        delete fileTree("${project.projectDir}/build/wpi-many-tests") {
+            include '**/.cannot-run-wpi'
+        }
     }
 
     doLast {
         // Run wpi-many.sh
-        exec {
-            commandLine 'bin/wpi-many.sh',
-                    '-i', "${project.projectDir}/tests/wpi-many/testin.txt",
-                    '-o', "${project.projectDir}/build/wpi-many-tests",
-                    '--', '--checker', 'nullness,interning,lock,regex,signature'
-        }
         def typecheckFilesDir = "${project.projectDir}/build/wpi-many-tests-results/"
+        try {
+            exec {
+                commandLine 'bin/wpi-many.sh',
+                        '-i', "${project.projectDir}/tests/wpi-many/testin.txt",
+                        '-o', "${project.projectDir}/build/wpi-many-tests",
+                        '-s',
+                        '--', '--checker', 'nullness,interning,lock,regex,signature'
+            }
+        } catch (Exception e) {
+            println("Failure: Running wpi-many.sh failed with a non-zero exit code.")
+            File wpiOut = new File("${typecheckFilesDir}/wpi-out")
+            if (wpiOut.exists()) {
+                println("========= Output from last run of wpi.sh: ========")
```

Comment:
I think it would be helpful to also state the file that contains this output (likewise for the similar code below).